ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts

```diff
@@ -265,6 +265,7 @@ class PlotLineOrBand {
             // apply defaults
             optionsLabel = merge({
                 align: horiz && isBand && 'center',
+                clip: false,
```

Comment:
Is this necessary? If we leave it out, it will become `undefined` by default which will still work correctly on line 350.